src/main/java/seedu/address/model/student/NameContainsKeywordsPredicate.java

```diff
@@ -9,7 +9,7 @@
  * Tests that a {@code Person}'s {@code Name} matches any of the keywords given.
  */
 public class NameContainsKeywordsPredicate implements Predicate<Student> {
-    private final List<String> keywords;
+    public final List<String> keywords;
```

Comment:
I changed to public because I created a method `getFindStudentDescriptorDetails` in a testingutil `StudentUtil` that required access to the keywords to create the command. Would it be better to use getters instead?

Reply:
Yeah I think your current implementation is fine then